model-optimizer/mo/middle/passes/mean_scale_values_test.py

```diff
@@ -38,137 +38,239 @@
                     'scaleshift_1_b': {'value': None, 'shape': None, 'kind': 'data'},
                     'scaleshift_1_data': {'value': None, 'shape': None, 'kind': 'data'},
                     'op_output': { 'kind': 'op', 'op': 'Result'},
-                    'op_output_1': { 'kind': 'op', 'op': 'Result'}
+                    'op_output_1': { 'kind': 'op', 'op': 'Result'},
+
+                    'const_mul': dict(kind='op', op='Const'),
+                    'const_mul_output': dict(kind='data'),
+                    'mul': dict(kind='op', op='Mul'),
+                    'mul_output': dict(kind='data'),
+
+                    'const_add': dict(kind='op', op='Const'),
+                    'const_add_output': dict(kind='data'),
+                    'add': dict(kind='op', op='Add'),
+                    'add_output': dict(kind='data'),
 
                     }
 
 
-class TestScaleShift_To_Preprocess(unittest.TestCase):
-    def test_move_scaleshift_to_preprocess_1(self):
+class TestMeanValues_To_Preprocess(unittest.TestCase):
+    def test_move_mean_values_to_preprocess_1(self):
         graph = build_graph(nodes_attributes,
                             [('placeholder_1', 'placeholder_1_data'),
-                             ('placeholder_1_data', 'scaleshift_1'),
-                             ('scaleshift_1', 'scaleshift_1_data'),
-                             ('scaleshift_1_w', 'scaleshift_1'),
-                             ('scaleshift_1_b', 'scaleshift_1'),
-                             ('scaleshift_1_data', 'op_output')
+                             ('placeholder_1_data', 'mul'),
+                             ('mul', 'mul_output'),
+                             ('const_mul', 'const_mul_output'),
+                             ('const_mul_output', 'mul'),
+                             ('mul_output', 'add'),
+                             ('add', 'add_output'),
+                             ('const_add', 'const_add_output'),
+                             ('const_add_output', 'add'),
+                             ('add_output', 'op_output'),
                              ],
                             {'placeholder_1_data': {'shape': np.array([1, 227, 227, 3])},
-                             'scaleshift_1_w': {'shape': np.array([3]), 'value': np.ones(3)},
-                             'scaleshift_1_b': {'shape': np.array([3]), 'value': np.array([-1, -2, -3])},
+                             'const_mul_output': {'shape': np.array([3]), 'value': np.ones(3)},
+                             'const_add_output': {'shape': np.array([3]), 'value': np.array([-1, -2, -3])},
```

Comment:
Please, add a test where the `const_add_output` node value has shape `[1,3,1,1]` instead of `[3]`